test/node/src/schema.test.ts

```diff
@@ -2427,6 +2427,28 @@ for (const dialect of DIALECTS) {
           })
         })
 
+        describe('add primary key constraint', async () => {
+          it('should add a primary key constraint', async () => {
+            const builder = ctx.db.schema
+              .alterTable('test')
+              .addPrimaryKeyConstraint('test_pkey', ['integer_col'])
+
+            testSql(builder, dialect, {
+              postgres: {
+                sql: 'alter table "test" add constraint "test_pkey" primary key ("integer_col")',
+                parameters: [],
+              },
+              mysql: {
+                sql: 'alter table `test` add constraint `test_pkey` primary key (`integer_col`)',
+                parameters: [],
+              },
+              sqlite: NOT_SUPPORTED,
+            })
+
+            await builder.execute()
+          })
+        })
```

Comment:
nitpicky

let's use a separate `if (dialect !== 'sqlite')` block for this.

Reply:
let's add a test case for 2 columns.

Reply:
done